packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx

```diff
@@ -51,9 +51,9 @@ export class TreeWidgetUiItemsProvider implements UiItemsProvider {
       const trees: SelectableContentDefinition[] = [];
 
       if (!this._treeWidgetOptions?.hideTrees?.modelsTree) {
-        trees.push({
-          label: TreeWidget.translate("modeltree"),
-          id: "model-tree",
+        const modelTree = {
```

Comment:
im thinking we  change this back, and just continue to push all the trees like before.

and add at the end, if a defaultTree is specified, we just move that up to the front

Reply:
Added the change to the  end for setting up defaultTree if user have setup